atef/tests/conftest.py

```diff
@@ -14,6 +14,23 @@
 CONFIG_PATH = TEST_PATH / "configs"
 
 
+def passive_checkout_configs() -> List[pathlib.Path]:
+    filenames = ['lfe.json', 'all_fields.json', 'ping_localhost.json']
+    config_paths = [CONFIG_PATH / fn for fn in filenames]
+    return config_paths
+
+
+def active_checkout_configs() -> List[pathlib.Path]:
+    filenames = ['active_test.json']
+    config_paths = [CONFIG_PATH / fn for fn in filenames]
+    return config_paths
+
+
+PASSIVE_CONFIG_PATHS = passive_checkout_configs()
+ACTIVE_CONFIG_PATHS = active_checkout_configs()
+ALL_CONFIG_PATHS = PASSIVE_CONFIG_PATHS + ACTIVE_CONFIG_PATHS
```

Comment:
For starters - this works just fine and doesn't need changing.

I wanted to point out that it can be nice to just make fixtures like so:
```python
@pytest.fixture(params=PASSIVE_CONFIG_PATHS)
def passive_config_path(request) -> pathlib.Path:
    return request.param
```

Then you just use `passive_config_path: pathlib.Path` as a test function argument and you don't have to repeat `pytest.mark.parametrize`

Reply:
pytest is a mysterious beast